app/utils.py

```diff
@@ -296,6 +296,17 @@ def validate_image(bytestring):
         return False
 
 
+def validate_django_image(uploaded_file):
+    try:
+        image = None
+        if uploaded_file:
+            image = images.Image(uploaded_file.read())
+            image.width
```

Comment:
this line looks weird. are you just making this call to validate that the image was constructed properly? if it wasn't, would we even reach this line?

Reply:
That's my assumption, yeah. This function is really just a Django variant on an existing function (validate_image, above); I added a note about that in the comment.